mysql/qrt/qrt.go

```diff
@@ -0,0 +1,68 @@
+package qrt
+
+import (
+	"sort"
+)
+
+// MysqlQrtBucket : https://www.percona.com/doc/percona-server/5.6/diagnostics/response_time_distribution.html
+// Represents a row from information_schema.Query_Reponse_Time
+type MysqlQrtBucket struct {
+	time  string
```

Comment:
Its actually a row from information_schema.QUERY_RESPONSE_TIME. which is a varchar :(